contribs/gmf/src/controllers/abstractmobile.js

```diff
@@ -97,6 +107,22 @@ gmf.AbstractMobileController = function(config, $scope, $injector) {
   goog.base(
       this, config, $scope, $injector);
 
+  // Automatically close the nav when a measure tool becomes active
+  $scope.$watch(goog.bind(function() {
```

Comment:
I would simply add the following to the button:
{{{
ng-click="mainCtrl.hideNav()"
}}}

Advantage (IMO): this will close the menu each time a button is clicked no matter if the tool is active or not.